src/extension.ts

```diff
@@ -699,15 +699,18 @@ export class Extension implements RunHooks {
           activeDecorations.push({ range: location.range });
       }
 
+      const decorationCount: Record<number, number> = {};
       const completedDecorations: Record<number, vscodeTypes.DecorationOptions> = {};
       for (const { location, duration } of completed) {
         if (uriToPath(location.uri) === editorPath) {
           const line = location.range.start.line;
+          const count = decorationCount[line] ?? 1;
+          decorationCount[line] = count + 1;
           completedDecorations[line] = {
             range: location.range,
             renderOptions: {
               after: {
-                contentText: ` \u2014 ${duration}ms`
```

Comment:
I remember we use a special character here: https://github.com/microsoft/playwright/blob/fd24521f2e11d0672852a7fe867bc3d01b8bcf94/packages/playwright-core/src/utils/stackTrace.ts#L148C41-L148C44

maybe we want to use it here as well?